src/features/game/types/expansions.ts

```diff
@@ -2750,6 +3199,422 @@ const DESERT_LAND_25_REQUIREMENTS: Requirements = {
   bumpkinLevel: 75,
 };
 
+const VOLCANO_LAND_6_REQUIREMENTS: Requirements = {
+  resources: {
+    Wood: 100,
+    Stone: 50,
+    Iron: 30,
+    Gold: 10,
+  },
+  sfl: 0,
+  coins: 0,
+  seconds: 5 * 60,
+  bumpkinLevel: 70,
+};
+
+const VOLCANO_LAND_7_REQUIREMENTS: Requirements = {
+  resources: {
+    Wood: 200,
+    Stone: 75,
+    Iron: 25,
+    Gold: 15,
+    Crimstone: 4,
+    Oil: 30,
```

Comment:
At least one Oil Reserve should be autoplaced before tile 7.  Otherwise, players will inevitably jump to Volcano "too ealry" and then be permanently stuck with no way to generate Oil.

Reply:
Cheers @brian-man The team have been discussing this - looks like 1 Oil will be placed on the initial volcano island